_data/simple-icons.json

```
@@ -7814,6 +7814,11 @@
             "hex": "00AF9C",
             "source": "https://github.com/topjohnwu/Magisk/blob/23ad611566b557f26d268920692b25aa89fc0070/app/src/main/res/drawable/ic_magisk.xml"
         },
+        {
+            "title": "Mahindra",
+            "hex": "FF0000",
+            "source": "https://www.mahindra.com/"
```

```suggestion
            "source": "https://www.mahindra.com"
```
Remove trailing slash